src/scss/components/ec-btn/_ec-btn.scss

```diff
@@ -0,0 +1,148 @@
+// Here we state the default value of each color, if we change a brand
+// we can override the default value with something else
+$ec-btn--primary-color: $info !default;
+$ec-btn--primary-color-hover: $info-hover !default;
+$ec-btn--secondary-color: $level-4-interactive-elements !default;
+$ec-btn--secondary-color-hover: $level-3-body-and-headings !default;
+$ec-btn--success-color: $success !default;
+$ec-btn--success-color-hover: $success-hover !default;
+$ec-btn--error-color: $error !default;
+$ec-btn--error-color-hover: $error-hover !default;
+$ec-btn--disabled-color: $level-6-disabled-lines !default;
+
+.ec-btn {
+  font-family: inherit;
+  font-size: inherit;
+  color: $white;
+  width: 100%;
+  max-width: 256px;
+  padding: 10px;
+  cursor: pointer;
+  transition: all 0.15s ease-out;
+  text-decoration: none;
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  line-height: 1.2;
+  text-align: center;
+  border: 1px solid transparent;
+  background-color: $ec-btn--secondary-color;
+
+  &--sm {
+    min-width: 96px;
+    padding: 5px 10px;
```

Comment:
I haven't seen multiplications of 5 in the entire spec, are you sure this is right?